apps/expo/src/app/index.tsx

```diff
@@ -5,6 +5,7 @@ import * as Notifications from "expo-notifications";
 import { Stack } from "expo-router";
 
 import registerForPushNotificationsAsync from "~/notifications/registerNotifications";
+import { logger } from "../../../../packages/logger";
```

Comment:
😢 for now probably better to actually use console.log on frontend, logger is meant to be used by only api for right now

Reply:
actually might be able to get it working on FE, but probably best to just use console.log for rn

Reply:
ok i will get this setup by sunday so that we can have nice logging on FE too, just wait a couple of days 🥲